pex/bin/pex.py

```diff
@@ -291,6 +291,17 @@ def configure_clp_pex_options(parser):
            'No value (alias for prefer, for backwards compatibility). '
            '[Default: %default]')
 
+  group.add_option(
+      '--include-pyc', '--no-include-pyc',
```

Comment:
`--compile` / `--no-compile` would match pip and is arguably more appropriate.

Reply:
Great suggestion! Thank you.